exatomic/adf/output.py

```diff
@@ -202,9 +202,36 @@ def parse_orbital(self):
             _re_orb_00: ['symmetry', 'vector', 'spin', 'occupation', 'energy', 'eV'],
             _re_orb_01: ['vector', 'occupation', 'energy', 'eV', 'dE']}
         key = _re_orb_00 if found[_re_orb_00] else _re_orb_01
-        start = stop = found[key][-1] + 5
-        while self[stop].strip(): stop += 1
-        df = self.pandas_dataframe(start, stop, cols[key])
+        start = stop = found[key][-1] + 4
+        starts = []
+        stops = []
+        irreps = []
+        while True:
```

Comment:
I know this is a very dangerous thing to do, but I couldn't find another way to test that there is no error raised when intentionally converting a non-int string to a string.

Reply:
It looks well bounded by `len(self)` so it's probably fine. Is line 214 necessary?

Reply:
It is so we can catch any `ValueError` when trying to convert a string to an int. This comes from the way ADF arranges the symmetry as blocks with the symmetry element in a new line. Small example:
```
A
       A-symmetry orbital stuff
T
       T symmetry orbital stuff
```